### PR TITLE
std::time fix typo on `now()` comments

### PR DESCRIPTION
**Pull request by bekicot**
Fix typo, update words, and remove some redundant word.

### CODE REVIEW

File: src/libstd/time.rs [hunk 1]
```diff
@@ -170,14 +170,14 @@ impl Instant {
         // * https://bugzilla.mozilla.org/show_bug.cgi?id=1487778 - a similar
         //   Firefox bug
         //
-        // It simply seems that this it just happens so that a lot in the wild
-        // we're seeing panics across various platforms where consecutive calls
+        // It seems that this is just happens a lot in the wild.
```
Comment (czipperz) [inline src/libstd/time.rs]: This grammar is still messed up.  Try removing the word `is`.

**Comment by kennytm** [comment]
r=me after  czipperz's comment is fixed.

**Comment by Centril** [comment]
Ping from triage @bekicot ^---

**Comment by bekicot** [comment]
@Centril Sorry for the delay. Currently having a very slow internet connection (on vacation). Cant clone the repository to do the required amend. Be back tomorrow and will resolve.

**Comment by jonas-schievink** [comment]
Triage ping @bekicot, this still needs the tiny change above

**Comment by joelpalmer** [comment]
@rustbot modify labels to +S-inactive-closed and -S-waiting-on-author

From Triage: @bekicot Closing due to inactivity. Please re-open before pushing changes. Thank you for the PR.

**Pull request closed without merging**